### PR TITLE
fix(core): Fix init for `AuditEventRelay`

### PR DESCRIPTION
**Pull request by ivov**
Followup to #9724

### CODE REVIEW

File: packages/cli/src/Server.ts [hunk 1]
```diff
@@ -252,6 +253,7 @@ export class Server extends AbstractServer {
 		// ----------------------------------------
 		const eventBus = Container.get(MessageEventBus);
 		await eventBus.initialize();
+		Container.get(AuditEventRelay).init();
```
Comment (tomi) [inline packages/cli/src/Server.ts:256]: Can this be injected in the constructor?

**Review by tomi — COMMENTED** [review]
What was the issue?

**Comment by ivov** [comment]
> What was the issue?

`AuditEventRelay` was not being initialized

> Can this be injected in the constructor?

That'd create a dep cycle

**Review by tomi — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.47.1`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.47.1)

**Pull request merged**